docs/section4/4-4.md

```diff
@@ -0,0 +1,34 @@
+# Airside Systems – Rule 4-4
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False  
+**Rule ID:** 4-4
+**Rule Description:** Where no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d, heating and/or cooling system fans shall simulated to be cycled ON and OFF to meet heating and cooling loads during occupied hours.  
+**Rule Assertion:** P-RMR = expected value                                             
+**Appendix G Section:** Section G3.1-4 Schedules exception #1 for the proposed building  
+**Appendix G Section Reference:** Section G3.1-10 HVAC Systems proposed column c and d  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+**Applicability Checks:**
+
+1. Applicable if no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d.
+
+**Function Call:** 
+
+1. get_proposed_hvac_modeled_with_virtual_heating()
+2. get_proposed_hvac_modeled_with_virtual_cooling()
+
+**Rule Logic:**  
+**Applicability Check 1:**
+- Call function to get list of applicable HVAC systems due to virtual cooling being modeled in the proposed per G3.1-10 d : `applicable_HVAC_systems_cooling_list_p = get_proposed_hvac_modeled_with_virtual_cooling(U_RMR,P_RMR)`
+- Call function to get list of applicable HVAC systems due to virtual heating being modeled in the proposed per G3.1-10 c:`applicable_HVAC_systems_heating_list_p = get_proposed_hvac_modeled_with_virtual_heating(U_RMR,P_RMR)`
+- Combine list of applicable hvac systems, there may be duplicates in the list of inapplicable hvac systems but it is not impactful: `applicable_HVAC_systems_list_p = applicable_HVAC_systems_cooling_list_p + applicable_HVAC_systems_heating_list_p`
+- For each HeatingVentilationAirAconditioningSystem in the P_RMR check if it this check is applicable: `for each hvac_p in P_RMR..HeatingVentilationAirAconditioningSystem:`
+    - Check if hvac_p is in the list of applicable systems, if it is found then set applicability flag to true: `if hvac_p in applicable_HVAC_systems_list_p: rule_applicability_flag = TRUE`
+    - If rule_applicability_flag = TRUE for hvac_p: `if rule_applicability_flag == TRUE:`
+        - Get operation during occupied periods: `operation_during_occupied_p = hvac_p.fan_systems[0].operation_during_occupied`  
+
+        **Rule Assertion:**  
+        - Case 1: For each applicable hvac system: `outcome == "UNDETERMINED" and raise_message "Check that heating or cooling system that is only being simulated in the P_RMR to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d are modeled to be cycled ON and OFF to meet heating and cooling loads during occupied hours."`
+    
```

Comment:
This rule was updated to be non-mandatory during the 2022.03.16 RDS  call due to the potential issues with heating and cooling having to be modeled with continuous or cycling during occupied hours (i.e. heating coil requires continuous and cooling coil requires cycling).